x-pack/test/functional/apps/rollup_job/hybrid_index_pattern.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import datemath from '@elastic/datemath';
+import expect from '@kbn/expect';
+import mockRolledUpData, { mockIndices } from './rollup_helpers';
+
+export default function ({ getService, getPageObjects }) {
+
+  const log = getService('log');
+  const es = getService('es');
+  const esArchiver = getService('esArchiver');
+  const retry = getService('retry');
+  const PageObjects = getPageObjects(['security', 'rollup', 'common', 'indexManagement', 'settings', 'discover']);
+
+  describe('rollup job', function () {
```

Comment:
Can we change this describe to something like `hybrid rollup job`?  There's already this one and that makes it harder to run an individual test suite or parse the results;
```
test/functional/apps/rollup_job/rollup_jobs.js:  describe('rollup job', function () {
```

Reply:
Fixed this. Side affect of copy pasta. :D